runtime/development/src/lib.rs

```diff
@@ -1492,10 +1492,17 @@ impl pallet_interest_accrual::Config for Runtime {
 	type Weights = ();
 }
 
+parameter_types! {
+	#[derive(scale_info::TypeInfo)]
+	pub const GeneralCurrencyPrefix: [u8; 12] = GENERAL_CURRENCY_INDEX_PREFIX;
+}
+
```

Comment:
Since this is a constant, I feel like it's better suited in the current place, e.g the `constants` module of `libs/primitives/lib.rs`. WDYT @mustermeiszer?

Reply:
Agreeing with William here. All constants should stay in the primitives or the respective const of libs